modular_skyrat/modules/modular_items/code/psych_cart.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
+	name = "\improper P.A.T cartridge"
+	desc = "Flaunt that Psychology Degree with this Cartridge!"
```

Comment:
Could be a bit more creative, but that's not necessarily a bad description either.
```suggestion
	desc = "Flaunt that Psychology degree with this cartridge!"
```